dev/com.ibm.websphere.appserver.features/visibility/public/mpReactiveStreams-1.0/com.ibm.websphere.appserver.mpReactiveStreams-1.0.feature

```diff
@@ -0,0 +1,18 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=com.ibm.websphere.appserver.mpReactiveStreams-1.0
+visibility=public
+singleton=true
+IBM-App-ForceRestart: install, \
+ uninstall
+IBM-ShortName: mpReactiveStreams-1.0
+Subsystem-Name: MicroProfile Reactive Streams 1.0
+IBM-API-Package: \
+  org.eclipse.microprofile.reactive.streams; type="stable"
```

Comment:
Do we need to expose `org.reactivestreams` as an API package as well?

Reply:
Added